src/include/btmem.h

```diff
@@ -1152,8 +1161,8 @@ struct __wt_update_value {
  */
 struct __wt_modify_vector {
     WT_SESSION_IMPL *session;
-    WT_UPDATE *list[WT_MODIFY_VECTOR_STACK_SIZE];
-    WT_UPDATE **listp;
+    WT_UPDATE_INSERT_TS list[WT_MODIFY_VECTOR_STACK_SIZE];
```

Comment:
While I'm doing the initial pass of updates in history store insertion, I need to keep track of what timestamp I should be inserting which is essentially going to be `min(upd->start_ts, prev_ts)`. So I've just hijacked this data structure since it's convenient.

Some alternatives would be to put this struct and its operations within a macro so you can create specific versions of the vector like so: `WT_VECTOR(WT_UPDATE*)` and `WT_VECTOR(WT_UPDATE_INSERT_TS)` kinda like the `TAILQ` stuff. Or of course `void*`.

Reply:
I'm reluctant to put more effort into the vector class, because I think we should move towards using more standard WiredTiger data structures if possible. aka: a hack is OK with me.